modules/system/classes/UpdateManager.php

```diff
@@ -429,7 +437,11 @@ public function seedModule($module)
         }
 
         $seeder = App::make($className);
-        $seeder->run();
+        $return = $seeder->run();
```

Comment:
Does this collect only seeder messages or all plugin migration file results of run() too?

Reply:
@LukeTowers just the seeders at the moment.

Reply:
Might as well make it for all of them

Reply:
@LukeTowers I've taken a look at what would be involved with allowing the migrations to defer messages too. The plugin migrations can be handled easily, but the module migrations are currently using `Illuminate\Database\Migrations\Migrator` to handle migrations. If we want the same functionality for them, we'll have to extend and augment that class. Is that something you would like done as well?

Reply:
We're fine with not having it available on module migrations

Reply:
Still want plugin migrations handled though